seed/views/columns.py

```diff
@@ -149,6 +149,20 @@ def retrieve(self, request, pk=None):
             'column': ColumnSerializer(c).data
         })
 
+    @ajax_request_class
+    @has_perm_class('can_modify_data')
+    def update(self, request, pk=None):
+        organization_id = request.query_params.get('organization_id', None)
+
+        request.data['shared_field_type'] = request.data['sharedFieldType']
+        del request.data['sharedFieldType']
```

Comment:
This could definitely be done via serializer, but I'd think that we should solve the dedup problem first in case this method gets heavily refactored.

Reply:
Documented as needing to change in the future.